src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -67,6 +68,9 @@ export interface FeedbackResponsesResponse {
 })
 export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
 
+  @ViewChild(QuestionSubmissionFormComponent)
+  submissionForm: QuestionSubmissionFormComponent | undefined;
```

Comment:
Same here:

Violation: Event should be used for parent-child communication

Ideally, there should be a event stream from the child component (that keep firing whether the input is valid or not) and here we just need to aggregate them (maybe `reducer` in rxjs).

Thought, honestly I am 100% sure it will works and need some research on this.

Reply:
Do you have any starting point I could go about to research this? I'm actually not sure how multiple optional event streams can be subscribed to much less aggregated